frontend/src/app/last-will/(editor)/[id]/succession/page.tsx

```diff
@@ -1,13 +1,27 @@
 'use client'
+import { useEffect } from 'react'
+import { Route } from '../../../../../components/ButtonsAndLinks/Route/Route'
 import { Headline } from '../../../../../components/Headline/Headline'
+import { routes } from '../../../../../services/routes/routes'
+import { useLastWillContext } from '../../../../../store/last-will/LastWillContext'
+import { SidebarPages } from '../../../../../types/sidebar'
 
 /**
  * Succession Page
  */
 const Succession = () => {
+	const { lastWill, services } = useLastWillContext()
```

Comment:
Remove because `'lastWill' is assigned a value but never used.`
```suggestion
	const { services } = useLastWillContext()
```